packages/itwinui-react/src/core/Table/Table.test.tsx

```diff
@@ -4761,18 +4005,6 @@ it('should render row with loading status', () => {
   expect(rows[1]).not.toHaveClass(`iui-loading`);
 });
 
-it('should navigate through table sorting with the keyboard', async () => {
```

Comment:
There don't seem to be any mocks here. So, curious why this was removed. Is it because sorting is tested in `e2e`?

Reply:
It was failing for some reason, so I moved it to e2e. 🤷